src/Raven.Studio/wwwroot/Content/css/styles-common-scoped.less

```diff
@@ -2157,6 +2157,14 @@ ul.properties {
     gap: @gutter-xs;
 }
 
+.align-items-center {
+    align-items: center;
+}
+
+.align-self-center {
+    align-self: center;
+}
+
```

Comment:
bs5 already provides that https://getbootstrap.com/docs/5.0/utilities/flex/#align-self ?

Reply:
ok this for bs3, so please ignore my comment